### PR TITLE
Android sign-in password/email

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** Android sign-in password/email
- **Is this PR related to a Monolith release?** Tuscany

### Resolves:

* [OKTA-413947](https://oktainc.atlassian.net/browse/OKTA-413947)

### CODE REVIEW

**Review by JayNewstrom — APPROVED** [review]

**Review by jaynewstrom-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/android/integrationsteps.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+Build a sign-in page for your app that captures both the username and the password.
+
+<div class="common-image-format">
+
+![Displays the Java SDK sign-in page](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-sign-on-screenshot-sign-in-java.png)
+
+</div>
+
+Begin the authentication process by calling Java SDK's [`IDXAuthenticationWrapper.begin()`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java#L603) method and getting a new [`ProceedContext`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/ProceedContext.java) object.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/android/integrationsteps.md:13]:
```suggestion
Begin the authentication process by calling the Java SDK's [`IDXAuthenticationWrapper.begin()`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java#L603) method and getting a new [`ProceedContext`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/ProceedContext.java) object.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/android/integrationsteps.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+After the user submits their credentials, call `IDXAuthenticationWrapper.authenticate()` with the credential values.
+
+```kotlin
+val authenticationResponse =
+     idxAuthenticationWrapper.authenticate(AuthenticationOptions(username, password)), proceedContext)
+```
+
+If the password is validated, the `IDXAuthenticationWrapper.authenticate()` method returns an [`AuthenticationResponse`](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/response/AuthenticationResponse.java) object with the following:
+
+1. `AuthenticationStatus` = `AWAITING_AUTHENTICATOR_SELECTION` <br>
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/android/integrationsteps.md:31]:
```suggestion
* `AuthenticationStatus= AWAITING_AUTHENTICATOR_SELECTION` &mdash; This status indicates that there are required authenticators that need to be verified.
```

Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/android/integrationsteps.md:31]: This formatting is just a suggestion. I've implemented the bullet/em-dash convention after the password/phone use case review with Chris (and have yet to fix all my previous use cases).

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/android/integrationsteps.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+     This status indicates that there are required authenticators that need to be verified.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/android/integrationsteps.md:32]:
```suggestion
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/android/integrationsteps.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+2. `Authenticators` = List of [authenticators](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/Authenticator.java) to be verified (in this case, there is only the email authenticator).
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-sign-in-pwd-email/android/integrationsteps.md:34]:
```suggestion
* `Authenticators` &mdash; List of [authenticators](https://github.com/okta/okta-idx-java/blob/master/api/src/main/java/com/okta/idx/sdk/api/client/Authenticator.java) to be verified (in this case, there is only the email authenticator).
```

**Review by vanngo-okta — APPROVED** [review]
Just a formatting suggestion. Otherwise, LGTM! 🚀

**Pull request merged**